internal/service/streamconnection/model_stream_connection_test.go

```diff
@@ -388,3 +409,15 @@ func tfConfigMap(t *testing.T, config map[string]string) types.Map {
 	}
 	return mapValue
 }
+
+func tfDBRoleToExecuteObject(t *testing.T, role, roleType string) types.Object {
```

Comment:
why does it start with tf?

Reply:
following convention from other methods. Also, it signifies we are working with Terraform model specifically. I believe we often follow this convention in our resources

Reply:
ah, ok, i suppose it's in the new plugin, thx